src/ostree/ot-builtin-reset.c

```diff
@@ -78,21 +78,29 @@ ostree_builtin_reset (int           argc,
     {
       if (opt_collection_id)
         {
+          const OstreeCollectionRef col_ref = { opt_collection_id, (char *)refspec };
```

Comment:
The `fixup reset` commit needs to be squashed (just adding this comment so that isn’t forgotten).